common/autoware_auto_perception_rviz_plugin/include/object_detection/object_polygon_display_base.hpp

```diff
@@ -127,6 +127,11 @@ class AUTOWARE_AUTO_PERCEPTION_RVIZ_PLUGIN_PUBLIC ObjectPolygonDisplayBase
     m_marker_common.addMessage(marker_ptr);
   }
 
+  void add_marker(visualization_msgs::msg::MarkerArray::ConstSharedPtr markers_ptr)
```

Comment:
@tkimura4 
Did you measure how much the visualization delay was shortened?
I'm just curious

Reply:
On my Desktop PC, The hz of rviz visualization does not decrease even when a number of objects exceeds 50.
So, I cannot measure the improvement of visualization delay.

Reply:
Cool, thanks!